proto/tendermint/store/types.proto

```diff
@@ -9,11 +9,14 @@ message BlockStoreState {
 }
 
 // TxInfo describes the location of a tx inside a committed block
-// as well as the result of executing the transaction.
+// as well as the result of executing the transaction and the raw log output.
 message TxInfo {
   int64 height    = 1;
   uint32 index    = 2;
   // The response code of executing the tx. 0 means
   // successfully executed, all others are error codes.
   uint32 code     = 3;
+  // The unprocessed log output generated during
+  // the execution of a transaction.
```

Comment:
[nit] to add some mention that this will be empty if transaction succeeded

```suggestion
  // The log output generated during the execution of a transaction.
  // Note: this is empty if the transaction succeeded.
```